eventtracking/backends/routing.py

```diff
@@ -134,3 +138,30 @@ def send_to_backends(self, event):
                 LOG.exception(
                     'Unable to send event to backend: %s', name
                 )
+
+
+class AsyncRoutingBackend(RoutingBackend):
+
+    def send(self, event):
+        """
+        Process the event using all registered processors and send it to all registered backends.
+
+        Logs and swallows all `Exception`.
+        """
+        try:
+            processed_event = self.process_event(event)
+        except EventEmissionExit:
+            return
+
+        for name, backend in six.iteritems(self.backends):
+            try:
+                pickled_backend = pickle.dumps(backend)
+                async_send.delay(name, pickled_backend, event)
+            except PicklingError:
+                LOG.exception(
+                    'Unable to pickle backend: %s', name
```

Comment:
name should be replace with `backend`

Reply:
Using `backend` will result in logs like this: `Unable to schedule task to send event to backend <eventtracking.backends.caliper.CaliperBackend object at 0x7f154cb5cac8>`
Is this acceptable?

Reply:
@danialmalik that's ugly then keep using `name`